gradle.properties

```diff
@@ -0,0 +1 @@
+gradleVersion=7.5.1
```

Comment:
Why does this exist? I dont like that the gradle version is now duplicated. Or is this generated by Gradle?

Reply:
Hmm, you're right. It is not generated by Gradle but it's referenced in `build.gradle`. I was going off [this repo](https://github.com/JetBrains/intellij-platform-plugin-template). I'm not sure there is a way around duplicating it :(

Reply:
Yeah, we don't need it because we don't use Dependabot. Removed in b628d5c